central/networkgraph/aggregator/aggregator_test.go

```diff
@@ -297,3 +298,24 @@ func TestAggregateExtConnsByName(t *testing.T) {
 	assert.Len(t, actual, len(expected))
 	protoassert.ElementsMatch(t, expected, actual)
 }
+
```

Comment:
Could we maybe add a unit test for `anonymizeIfNoExternalIPs` for the sake of completeness?

Reply:
Small test added. It covers entities with a different type, and external entities that are not 'discovered'.